packages/snaps-cli/src/cmds/build/index.ts

```diff
@@ -16,8 +16,17 @@ export = {
       .option('src', builders.src)
       .option('stripComments', builders.stripComments)
       .option('transpilationMode', builders.transpilationMode)
+      .option('depsToTranspile', builders.depsToTranspile)
       .option('writeManifest', builders.writeManifest)
-      .implies('writeManifest', 'manifest');
+      .implies('writeManifest', 'manifest')
+      .implies('depsToTranspile', 'transpilationMode')
+      .middleware((argv) => {
+        if (argv.depsToTranspile && argv.transpilationMode !== 'localAndDeps') {
+          throw Error(
+            '"transpilationMode" must be "localAndDeps" in order to transpile dependencies.',
```

Comment:
Thought of one further improvement here:

```suggestion
            '"depsToTranspile" can only be specified if "transpilationMode" is set to "localAndDeps" .',
```